core/ez/config.py

```diff
@@ -0,0 +1,81 @@
+from dataclasses import dataclass
+import os
+from pathlib import Path
+import yaml
+
+
+class Config:
+    def __init__(self, config_file):
+        self.config_file = Path(config_file)
+        if not self.config_file.is_file():
+            raise FileNotFoundError(f"Config file '{self.config_file}' not found.")
+        self.config = yaml.safe_load(open(self.config_file))
+        self.database = self._load_database_config()
+
+    def reload(self):
+        if hasattr(self, "config"):
+            for key in list(self.env):
+                os.environ.pop(key, None)
+        self.config = yaml.safe_load(open(self.config_file))
+        for key, value in self.env.items():
+            os.environ[key] = value
+
+        self._load_database_config()
+
+    def __getattr__(self, name):
+        return self.config[name]
```

Comment:
Did you test it?

Reply:
This one no, I had no use to it.
I forgot to remove it.